Solutions/Microsoft 365 Defender/Hunting Queries/Delivered Bad Emails from Top bad IPv4 addresses .yaml

```diff
@@ -0,0 +1,18 @@
+id: cdac93ef-56c0-45bf-9e7f-9cbf0ad06567
+name: Determines emails sent by top malicious/bad IP addresses.
+description: |
+  This query determines emails sent by top malicious/bad IP addresses. Arbitrary cutoff assumed as 5, increase or decrease as needed 
+requiredDataConnectors:
+- connectorId: OfficeATP
+  dataTypes:
+  - EmailEvents
+tactics:
+- Initial access
+query: |
+  let cutoff = 5;
+  EmailEvents
+  | where ThreatTypes has "Malware" or ThreatTypes has "Phish" 
+  | summarize count() by SenderIPv4 
+  | where count_ > cutoff // Arbitrary cutoff, increase or decrease as needed 
+  | join EmailEvents on SenderIPv4  
+  | where DeliveryAction == "Delivered"
```

Comment:
Can we change this too =~ for case sensitivity.

Reply:
done